Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/SelectorsPollerScript.js

```diff
@@ -47,7 +47,8 @@ window.__firefox__.execute(function($) {
   let notYetQueriedIds = []
   
   const CC = {}
-  CC.allSelectorsToRules = CC.allSelectorsToRules || new Map()
+  CC.allHideSelectorsToRules = CC.allHideSelectorsToRules || new Map()
```

Comment:
Why is this necessary? `CC = {}` on line 49. So it's clear that `CC` does NOT have an `allHideSelectorsToRules` property.
So therefore it will always hit the `||` condition and initialize to `new Map()`.
Might as well do: `CC.allHideSelectorsToRules = new Map()`

Reply:
It's taken from the typescript file which works a little bit differently on desktop. I thought about cleaning all this up but I decided not to touch the file too much because I'd still like to use the same source file for both iOS and desktop. But maybe now that these files are sort of going in separate directions I should.